mountd/mountd.c

```diff
@@ -417,7 +417,8 @@ int mountd_main(int argc, char **argv)
 			global_conf.tcp_port = cp_get_group_kv_long(optarg);
 			break;
 		case 'n':
-			nodetach = !optarg ?: cp_get_group_kv_long(optarg);
+			if (optarg)
+				nodetach = cp_get_group_kv_long(optarg);
```

Comment:
@neheb, @namjaejeon,

With this change, `-n` does nothing and so nodetach mode is broken.
The previous behavior, which is documented, is that `-n` should be equivalent to `-n1`.

Reply:
Okay, I will remove this line in the patch.